### ISSUE TITLE
Windows Packaging: qt platform dlls missing

### ISSUE THREAD

**Issue by MartinOehler** [openconnect/openconnect-gui#115]
Hello,

here is a follow-up to #114. The NSIS package is now built but it is missing the Qt platform plugin for windows. Comparing the result with the installer archive of the openconnect-1.3 release, the following files which are necessary to start a Qt application on a system without Qt installed are missing:

```
platforms/qminimal.dll
platforms/qoffscreen.dll
platforms/qwindows.dll
```

At least the missing `qwindows.dll` really hurts, it leads to the infamous *platform plugin missing* error.
Executing `windeployqt` correctly collects these files. Without them the package is not distributable.

Thanks,
Martin

P.S.: checking the output of `7z l openconnect-installer.exe` (v1.3) and the product of the current mingw build shows some more differences. Perhaps you might have a look at it.

**Comment by horar** [comment]
Yeap, you are building devel version... there are few more issues to solve (or just to submit) in next days...

**Comment by MartinOehler** [comment]
Having proper build instructions makes it much easier to contribute to a project. If you need help writing them down we could do this within #84. But without having a reproducible build it's not that easy.

**Comment by horar** [comment]
Should be resolved by 78bf6ae5e36c618d182020df5486ebbf3242dd01
